src/deploy/functions/release/fabricator.ts

```diff
@@ -519,6 +541,15 @@ export class Fabricator {
     }
   }
 
+  async registerBlockingTrigger(
+    endpoint: backend.Endpoint & backend.BlockingTriggered,
+    update: boolean
+  ): Promise<void> {
+    await this.executor
+      .run(() => authBlocking.registerTrigger(endpoint, update))
```

Comment:
Again, this is only appropriate if it's an auth blocking function.

Reply:
Changed it to be
```
await this.executor
      .run(() => services.serviceForEndpoint(endpoint).registerTrigger(endpoint, update))
      .catch(rethrowAs(endpoint, "register blocking trigger"));
```